addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -3,6 +3,7 @@ odoo.define('web_editor.convertInline', function (require) {
 
 var FieldHtml = require('web_editor.field.html');
 
+const SELECTORS_IGNORE = /(^\*$|:hover|:before|:after|:active|:link|::|'|\([^(),]+[,(])/g
```

Comment:
- There is a missing `;` at the end of that line, your linter should explode! :wink: 
- I am not sure to get the `[,(]` at the end of the regex, is it to ignore all nested rules with parenthesis? Should not we allow `:not(:nth-child(3))`?

Reply:
> I am not sure to get the [,(] at the end of the regex, is it to ignore all nested rules with parenthesis? Should not we allow :not(:nth-child(3))?

The problem is mostly that this is a single regex, so if we match until a closing parenthesis then we would fail to match

    :not(:nth-child(3), .foo)

which would allow that rule through, which would then break. So I took the option to just reject nested pseudo-classes, of which we seem to have none anyway.

Reply:
Seems legit :+1:

Reply:
I'll add that to the commit message though, so we have a record of why the regex is that way.